android/jni/app-android.cpp

```diff
@@ -227,6 +232,36 @@ void AndroidLogger::Log(const LogMessage &message) {
 	}
 }
 
+bool Android_IsContentUri(const std::string &filename) {
+	return startsWith(filename, "content://");
+}
+
+int Android_OpenContentUriFd(const std::string &filename) {
+	if (!nativeActivity) {
+		return -1;
+	}
+	auto env = getEnv();
+	jstring param = env->NewStringUTF(filename.c_str());
+	int fd = env->CallIntMethod(nativeActivity, openContentUri, param);
+	return fd;
+}
+
+void Android_CloseContentUriFd(int fd) {
+	if (!fd) {
```

Comment:
nit: Unused anyway.  Seems like we don't need to close any special way?

-[Unknown]

Reply:
Right, first I thought we needed to, then I discovered "detachFd". This got left behind in the shuffle.